### PR TITLE
Fix new flake8 from pyflakes 2.2.0 release

### PR DESCRIPTION
**Pull request by rooftopcellist**
##### SUMMARY
Zuul uses flake8, which passes (locally as well), but people still use make flake8 , which has a different pyflakes version (inside the awx venv).

```
3.7.9 (mccabe: 0.6.1, pycodestyle: 2.5.0, pyflakes: 2.1.1) CPython 3.6.8 on Linux
bash-4.4$ which flake8
/usr/local/bin/flake8
bash-4.4$ source /venv/awx/bin/activate
(awx) bash-4.4$ flake8 --version
3.7.9 (mccabe: 0.6.1, pycodestyle: 2.5.0, pyflakes: 2.2.0) CPython 3.6.8 on Linux
(awx) bash-4.4$ which flake8
/venv/awx/bin/flake8
```

`pyflakes` released a new version (2.2.0) on April 9th - https://pypi.org/project/pyflakes/
So currently, if you run make flake8 it will give you a bunch of new failures which zuul doesn't get

##### ISSUE TYPE

 - Bugfix Pull Request

##### COMPONENT NAME

 - API

##### AWX VERSION

```
11.0.0
```

### CODE REVIEW

File: awx/main/analytics/collectors.py [hunk 1]
```diff
@@ -309,6 +309,6 @@ def _copy_table(table, query, path):
                                  main_unifiedjobtemplate.status 
                                  FROM main_unifiedjobtemplate, django_content_type
                                  WHERE main_unifiedjobtemplate.polymorphic_ctype_id = django_content_type.id
-                                 ORDER BY main_unifiedjobtemplate.id ASC) TO STDOUT WITH CSV HEADER'''.format(since.strftime("'%Y-%m-%d %H:%M:%S'"))    
+                                 ORDER BY main_unifiedjobtemplate.id ASC) TO STDOUT WITH CSV HEADER'''
```
Comment (rooftopcellist) [inline awx/main/analytics/collectors.py:312]: @benthomasson is aware of this.  He said it's better we don't filter Unified Job Templates by the last collection time anyways.

File: awx/main/dispatch/worker/callback.py [hunk 2]
```diff
@@ -91,7 +91,7 @@ def flush(self, force=False):
                     for e in events:
                         try:
                             if (
-                                isinstance(exc, IntegrityError),
+                                isinstance(exc, IntegrityError) or
```
Comment (rooftopcellist) [inline awx/main/dispatch/worker/callback.py]: @ryanpetrello Is there any particular reason for the `comma` syntax here?  I tested this out and `or` is functionally the same.  Just wanted to check as you wrote this and it is some mission critical code.

File: awx/api/serializers.py [hunk 3]
```diff
@@ -3668,7 +3668,7 @@ def validate(self, attrs):
                         attrs.get('survey_passwords', {}).pop(key, None)
                     else:
                         errors.setdefault('extra_vars', []).append(
-                            _('"$encrypted$ is a reserved keyword, may not be used for {var_name}."'.format(key))
+                            _('"$encrypted$ is a reserved keyword, may not be used for {}."'.format(key))
```
Comment (ryanpetrello) [inline awx/api/serializers.py:3671]: Let's just change these to f strings while we're here :D

File: awx/conf/fields.py [hunk 4]
```diff
@@ -172,9 +172,9 @@ def run_validators(self, value):
                         netloc = '{}:{}'.format(netloc, url_parts.port)
                     if url_parts.username:
                         if url_parts.password:
-                            netloc = '{}:{}@{}' % (url_parts.username, url_parts.password, netloc)
+                            netloc = '{}:{}@{}'.format(url_parts.username, url_parts.password, netloc)
```
Comment (ryanpetrello) [inline awx/conf/fields.py:175]: Same comment, let's get rid of of `.format` since we're here 🤷

File: awx/main/analytics/collectors.py (hunk 1 above)
Comment (ryanpetrello) [reply awx/main/analytics/collectors.py:312]: Ouch, this is actually a bug.  Is this something that needs backporting?

File: awx/main/dispatch/worker/callback.py (hunk 2 above)
Comment (ryanpetrello) [reply awx/main/dispatch/worker/callback.py]: No, this is a bug, and it should be `and`.  I probably meant to wrap this in `if all()` or something, let's change this to `and`.

File: awx/main/tests/factories/tower.py [hunk 5]
```diff
@@ -220,7 +220,7 @@ def create_job_template(name, roles=None, persisted=True, webhook_service='', **
     if 'organization' in kwargs:
         org = kwargs['organization']
         if type(org) is not Organization:
-            org = mk_organization(org, '%s-desc'.format(org), persisted=persisted)
+            org = mk_organization(org, '{}'.format(org), persisted=persisted)
```
Comment (ryanpetrello) [inline awx/main/tests/factories/tower.py]: Why not just `(org, org, persisted=persisted)`?

File: awx/main/tests/factories/tower.py [hunk 6]
```diff
@@ -298,7 +298,7 @@ def create_organization(name, roles=None, persisted=True, **kwargs):
     labels = {}
     notification_templates = {}
 
-    org = mk_organization(name, '%s-desc'.format(name), persisted=persisted)
+    org = mk_organization(name, '{}'.format(name), persisted=persisted)
```
Comment (ryanpetrello) [inline awx/main/tests/factories/tower.py]: Same here, `(name, name, ...)`

File: awx/main/tests/factories/tower.py (hunk 5 above)
Comment (rooftopcellist) [reply awx/main/tests/factories/tower.py]: Yeah, that would work too.

File: awx/main/analytics/collectors.py (hunk 1 above)
Comment (rooftopcellist) [reply awx/main/analytics/collectors.py:312]: Nope. Functionally this changes nothing.  @ryanpetrello I talked this through with @benthomasson this morning, he is aware of the fact that this doesn't filter by time and instead sends all unified job templates.  He said it is actually better that way.

**Review by AlanCoding — APPROVED** [review]

**Pull request merged**